### ISSUE TITLE
Transliteration and reverse transliteration yield different output

### ISSUE THREAD

**Issue by malayaleecoder** [varnamproject/libvarnam#134]
On transliterating `varnamproject` , `വർണംപ്രൊജെക്റ്റ് ` is obtained. But on reverse transliterating `വർണംപ്രൊജെക്റ്റ് `, the output seen is `varnamprojekt`. Have tested with other common words(which yield the correct results). Not sure if this issue is specific to this word.

**Comment by lonesword** [comment]
Did you get this result before or after training on a word corpus? If already trained, transliteration can produce the intended result even when an approximate term is provided. But in reverse transilteration, it probably outputs the absolute phonetic match.

**Comment by navaneeth** [comment]
Yeah. Reverse transliteration does absolute phonetic match. It doesn’t consider the learning data. 

---
Navaneeth

**Comment by lonesword** [comment]
In that case can we close this issue?
